src/TextField/EnhancedTextarea.js

```diff
@@ -56,9 +56,15 @@ class EnhancedTextarea extends Component {
   };
 
   state = {
-    height: this.props.rows * rowsHeight,
+    height: rowsHeight,
```

Comment:
@oliviertassinari @nathanmarks I had a good chuckle when I read this haha

I've updated the PR so anything without an explicit initial value is `null`